app/popup-init.html

```diff
@@ -25,7 +25,7 @@
         * page may be a further UX improvement.
         */
       html {
-        width: 357px;
+        width: 400px;
```

Comment:
No remaining instances of 357 that effects the width of the extension

https://github.com/user-attachments/assets/fccfd4be-71cd-4193-ac75-37aaa971b618